### PR TITLE
Fix #1 by checking version of OpenNebula server

### PR DESCRIPTION
**Pull request by fleXible**
Before instantiating any Host or VirtualNetwork object, check if server is running OpenNebula 5 and change the class attributes accordingly.

### CODE REVIEW

**Comment by sysradium** [comment]
@fleXible 
Thanks for the thinks. Can you please run the tests locally and correct mock assertions accordingly? Probably everything is fine, you just need to bump mock call counts.

File: oca/host.py [hunk 1]
```diff
@@ -78,6 +78,13 @@ def allocate(client, hostname, im, vmm, tm, cluster_id=-1):
         return host_id
 
     def __init__(self, xml, client):
+        # remove 'vn_mad' attribute in OpenNebula >= 5
+        xml_types = Host.XML_TYPES
+        if client.one_version is None:
+            client.version()
+        if client.one_version.startswith('5.') and 'vn_mad' in xml_types:
```
Comment (fleXible) [inline oca/host.py]: I just realized, this is not the smartest way to check the version, as it will break with version 6.

**Comment by fleXible** [comment]
@sysradium 
The PR is complete from my point of view. Would you be so kind and merge it?

**Comment by sysradium** [comment]
@fleXible 👍

**Comment by fleXible** [comment]
I just realized, forgot to update setup.py and bump the version number.

**Pull request merged**